### PR TITLE
Remove warning message that regarding unknown event type

### PR DESCRIPTION
**Pull request by timl3136**
**What changed?**
Remove warning message for unknown event type

**Why?**
It's creating too much warning log that overwhelm system

**How did you test it?**
Unit test

**Potential risks**
Only remove logger so no danger

### CODE REVIEW

File: internal/internal_utils.go [hunk 1]
```diff
@@ -469,9 +469,7 @@ func estimateHistorySize(logger *zap.Logger, event *s.HistoryEvent) int {
 			sum += len(event.SignalExternalWorkflowExecutionInitiatedEventAttributes.Input)
 		}
 
-	default:
-		logger.Warn("unknown event type", zap.String("Event Type", event.GetEventType().String()))
-		// Do not fail to be forward compatible with new events
+	default: // Do not fail to be forward compatible with new events
```
Comment (taylanisikdemir) [inline internal/internal_utils.go:472]: you can remove default case since it's no-op

**Review by taylanisikdemir — APPROVED** [review]

File: internal/internal_utils.go (hunk 1 above)
Comment (timl3136) [reply internal/internal_utils.go]: Done

File: internal/internal_utils.go [hunk 2]
```diff
@@ -468,10 +468,6 @@ func estimateHistorySize(logger *zap.Logger, event *s.HistoryEvent) int {
 			sum += len(event.SignalExternalWorkflowExecutionInitiatedEventAttributes.Control)
 			sum += len(event.SignalExternalWorkflowExecutionInitiatedEventAttributes.Input)
 		}
-
-	default:
-		logger.Warn("unknown event type", zap.String("Event Type", event.GetEventType().String()))
```
Comment (3vilhamster) [inline internal/internal_utils.go:473]: Maybe it could be useful to have this log with Debug level?

Comment (timl3136) [reply internal/internal_utils.go:473]: Good idea, let me change that to debug level logs

**Pull request merged**